lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -66,10 +67,14 @@ def build_index(self, server_config, dates):
 
         return f"/{index}"
 
-    def build_index_from_metadata(self):
+    def build_index_from_metadata(self, root_index_name:str) -> str:
+        """Args
+            - root_index_name: Retrieve the list of ES indices from
+                the metadata table based on a given root_index_name.
```

Comment:
The description here should really be the description of the *method* rather than of the parameter. Maybe:

```
    """
    Retrieve the list of ES indices from the dataset index map metadata based on a given root index name.

    Args:
        root_index_name: root index name

    Returns:
        An Elasticsearch query URL string listing the set of indices containing documents for the specified
        root index name.
    """
```